### PR TITLE
Initial Receipt API implementation

### PR DESCRIPTION
**Pull request by Kanokorn**

### CODE REVIEW

File: operations/receipt.go [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+//	if e := client.Do(receipts, list); e != nil {
+//		panic(e)
+//	}
+//
+//	fmt.Println("receipts #100-#120:", receipts.Data)
+//
+type ListReceipts struct {
+	List
+}
+
```
Comment (iporsut) [inline operations/receipt.go]: Could you add support JSON payload? 

e.g. look in `operations/schedule.go`

```go
type ListSchedules struct {
	List
}

func (req *ListSchedules) MarshalJSON() ([]byte, error) {
	return json.Marshal(req.List)
}

func (req *ListSchedules) Op() *internal.Op {
	return &internal.Op{
		Endpoint:    internal.API,
		Method:      "GET",
		Path:        "/schedules",
		ContentType: "application/json",
	}
}
```

Comment (Kanokorn) [inline operations/receipt.go]: 👌

Comment (Kanokorn) [inline operations/receipt.go]: Updated ka

File: receipt.go [hunk 2]
```diff
@@ -0,0 +1,30 @@
+package omise
+
+import "time"
+
+// Receipt represents Omise's receipt object.
+// See https://www.omise.co/receipts-api for more information.
```
Comment (pitiphong-p) [inline receipt.go]: The URL is wrong it should be `https://www.omise.co/receipts-api`

File: testdata/fixtures/api.omise.co/receipts-get.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
+{
+    "object": "list",
+    "from": "2017-07-01T00:00:00Z",
+    "to": "2017-07-31T23:59:59Z",
+    "offset": 0,
+    "limit": 20,
+    "total": 3,
```
Comment (pitiphong-p) [inline testdata/fixtures/api.omise.co/receipts-get.json]: Not sure if this is intended but the `total` number is 3 but you have only 1 receipt object in the `data` list

Comment (Kanokorn) [inline testdata/fixtures/api.omise.co/receipts-get.json]: Updated ka

File: receipt.go (hunk 2 above)
Comment (pitiphong-p) [inline receipt.go]: Sorry I also put the wrong URL in the comment. It should be `https://www.omise.co/receipt-api`

Comment (Kanokorn) [inline receipt.go]: will fix ka 🙇

**Pull request merged**